### PR TITLE
Feature/fixing frontend

### PR DESCRIPTION
**Pull request by jeppekroghitk**

### CODE REVIEW

File: assets/src/components/calendar-header.js [hunk 1]
```diff
@@ -41,47 +41,47 @@ function CalendarHeader({ date, setDate }) {
...
-              >
-                {dayjs(date).format("D. MMMM YYYY")}
-                <span className="small">X</span>
+              <label htmlFor="calendar-datepicker" className="datepicker-label">
+                <input
+                  type="date"
+                  id="calendar-datepicker"
+                  min={dayjs(new Date()).format("YYYY-MM-DD")}
+                  value={dayjs(date).format("YYYY-MM-DD")}
+                  onChange={(e) => onChangeDate(e)}
```
Comment (tuj) [inline assets/src/components/calendar-header.js]: no need for wrapping in anonymous function
```
onChange={onChangeDate}
```

File: assets/src/components/calendar-selection-box.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+/**
+ * Calendar selection box component.
+ *
+ * @param {object} props Props.
+ * @param {object} props.calendarSelection Object containing selection info
+ *   returned by fullcalendar
+ * @returns {object} Calendar selection box
+ */
+function calendarSelectionBox({ calendarSelection }) {
+  useEffect(() => {}, []);
```
Comment (tuj) [inline assets/src/components/calendar-selection-box.js]: remove

File: assets/src/components/calendar-selection-box.js [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+/**
+ * Calendar selection box component.
+ *
+ * @param {object} props Props.
+ * @param {object} props.calendarSelection Object containing selection info
+ *   returned by fullcalendar
+ * @returns {object} Calendar selection box
+ */
+function calendarSelectionBox({ calendarSelection }) {
```
Comment (tuj) [inline assets/src/components/calendar-selection-box.js]: rename to CalendarSelectionBox

File: assets/src/components/info-box.js [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+  const [infoBoxHeader, setInfoBoxHeader] = useState("");
+  const [infoBoxContent, setInfoBoxContent] = useState("");
+  const [showInfoBox, setShowInfoBox] = useState("flex");
+
+  useEffect(() => {
+    setInfoBoxColor(config.info_box_color);
+    setInfoBoxHeader(config.info_box_header);
+    setInfoBoxContent(config.info_box_content);
+  }, [config]);
+  const hideInfoBox = () => {
```
Comment (tuj) [inline assets/src/components/info-box.js]:
```
const hideInfoBox = () => setShowInfoBox("none");
```

File: assets/src/components/info-box.js [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+import "./info-box.scss";
+
+/**
+ * Information box component.
+ *
+ * @param {object} props Props.
+ * @param {object} props.config Object containing configuration from drupal
+ * @returns {object} Info box component
+ */
+function infoBox({ config }) {
```
Comment (tuj) [inline assets/src/components/info-box.js]: Rename to InfoBox

**Review by tuj — CHANGES_REQUESTED** [review]

File: assets/src/app.scss [hunk 6]
```diff
@@ -40,7 +64,99 @@ body {
   from {
     opacity: 0;
   }
+
   to {
     opacity: 1;
   }
 }
+
```
Comment (martinyde) [inline assets/src/app.scss]: Move to calendar scss or seperate calendar-selection.scss

**Review by tuj — APPROVED** [review]

**Pull request merged**